mithril-aggregator/src/database/query/signed_entity/get_signed_entity.rs

```diff
@@ -1,12 +1,13 @@
 use sqlite::Value;
 
-use mithril_common::entities::SignedEntityTypeDiscriminants;
+use mithril_common::entities::{Epoch, SignedEntityTypeDiscriminants};
 use mithril_common::StdResult;
 use mithril_persistence::sqlite::{Query, SourceAlias, SqLiteEntity, WhereCondition};
 
 use crate::database::record::SignedEntityRecord;
 
 /// Simple queries to retrieve [SignedEntityRecord] from the sqlite database.
+#[derive(Debug, PartialEq)]
```

Comment:
Why is it needed, tests should probably be done on real data here?

Reply:
This was removed with the new specific implementation for retrieving `CardanoStakeDistribution` signed entity record by epoch.